### PR TITLE
Welcome notificaiton

### PR DESCRIPTION
**Pull request by svendjanis**
#21

### CODE REVIEW

File: client/apps/shop/src/app/shared/components/login-signup-dialog/login-signup-dialog.component.ts [hunk 1]
```diff
@@ -87,7 +90,15 @@ export class LoginSignupDialogComponent extends RxDestroy implements OnInit {
           .set({
             createdOn: Date.now()
           })
-          .then()
+          .then(() => {
```
Comment (flauc) [inline client/apps/shop/src/app/shared/components/login-signup-dialog/login-signup-dialog.component.ts]: We should rewrite this as an observable and use the `notify()` operator

File: client/apps/shop/src/app/shared/components/login-signup-dialog/login-signup-dialog.component.ts [hunk 2]
```diff
@@ -87,7 +90,15 @@ export class LoginSignupDialogComponent extends RxDestroy implements OnInit {
...
+            this.snackBar.open(
+              'You have successfully created an account',
+              'Dismiss',
+              {
+                duration: 2500
+              }
+            );
+          })
           .catch();
```
Comment (flauc) [inline client/apps/shop/src/app/shared/components/login-signup-dialog/login-signup-dialog.component.ts]: We also need a notification for a failed request

Comment (svendjanis) [reply client/apps/shop/src/app/shared/components/login-signup-dialog/login-signup-dialog.component.ts]: When user enters wrong combination of email and password, he will already got mat-error, should it be notification also?

Comment (flauc) [reply client/apps/shop/src/app/shared/components/login-signup-dialog/login-signup-dialog.component.ts]: No you're right.

**Pull request merged**